mlrun/feature_store/feature_vector.py

```diff
@@ -407,3 +408,19 @@ def to_csv(self, target_path, **kw):
         """return results as csv file"""
         size = CSVTarget(path=target_path).write_dataframe(self._merger.get_df(), **kw)
         return size
+
+
+class FixedWindowType(Enum):
+    CurrentOpenWindow = 1
+    LastClosedWindow = 2
+
+    def to_qbk_fixed_window_type(self):
+        try:
+            from storey import FixedWindowType as QueryByKeyFixedWindowType
+        except ImportError as exc:
+            raise ImportError(f"storey not installed, use pip install storey, {exc}")
+        if self == FixedWindowType.LastClosedWindow:
+            return QueryByKeyFixedWindowType.LastClosedWindow
+        elif self == FixedWindowType.CurrentOpenWindow:
+            return QueryByKeyFixedWindowType.CurrentOpenWindow
+        return None
```

Comment:
If some will add a new FixedWindowType and forget to handle it in this function I want it to explode not to fail quietly (causing hard to debug bugs)
```suggestion
        else:
            raise NotImplementedError(
                f"Provided fixed window type is not supported. fixed_window_type={self}"
            )
```